README.md

```diff
@@ -87,6 +90,29 @@ module.exports.options = {
 }
 ```
 
+#### fastify version discovery
+
+If fastify has been installed local to plugin.js that copy will be used instead of the global copy included with fastify-cli.
+
+### generate
+
+`fastify-cli` can also help with generating some project scaffolding to
+kickstart the development of your next fastify application. To use it:
+
+1. `mkdir yourapp`
+2. `cd yourapp`
+3. `npm init`
+4. `fastify generate`
+5. `npm install`
+
+The sample code offers you three npm tasks:
+
+* `npm start` - starts the application
+* `npm run colada` - starts the application with
```

Comment:
Would be cool something like `npm start --pretty`.

Reply:
this is something we should check again in the future `npm start -- --pretty` _could_ work.